tutorials/quantum_oracle/quantum_oracle_tutorial.ipynb

```diff
@@ -0,0 +1,183 @@
+{
```

Comment:
<p>I think it's a good idea to remind/ explain the reader that <code>^=</code> is bitwise XOR of the binary representation of <code>res</code>  and  &nbsp;<code>x%2 == 1</code> and assigning the result on <code>res</code>.</p><p>I would also consider explaining again shortly what <code>x %2 == 1</code> does (oddity check).</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Classiq/classiq-library/pull/186/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>